Dockerfile

```diff
@@ -3,8 +3,7 @@
 FROM python:3.5.2-alpine
 MAINTAINER Regner Blok-Andersen <shadowdf@gmail.com>
 
-ADD main.py /app/
-ADD requirements.txt /app/
+ADD . /app/
```

Comment:
May I ask why you prefer to add everything to the container instead of specifying? Always worry about adding a bunch of garbage and bloating the size of the image.